docs/standard/generics.md

```diff
@@ -49,18 +49,20 @@ Here is a small program that illustrates the efficiency of knowing the data stru
   }
 ```
 
-This program yields the following output:
+This program yields an output similar to the following:
 
 ```console
-Generic Sort: System.Collections.Generic.List\`1[System.Int32] Time taken: 0.0789ms
-Non-Generic Sort: System.Collections.ArrayList Time taken: 2.4324ms
+Generic Sort: System.Collections.Generic.List`1[System.Int32]
+ Time taken: 0.0034ms
+Non-Generic Sort: System.Collections.ArrayList
+ Time taken: 0.2592ms
 ```
 
 The first thing you notice here is that sorting the generic list is significantly faster than for the non-generic list. You might also notice that the type for the generic list is distinct ([System.Int32]) whereas the type for the non-generic list is generalized. Because the runtime knows the generic `List<int>` is of type int, it can store the list elements in an underlying integer array in memory while the non-generic `ArrayList` has to cast each list element as an object as stored in an object array in memory. As shown through this example, the extra castings take up time and slow down the list sort.
 
-The last useful thing about the runtime knowing the type of your generic is a better debugging experience. When you are debugging a generic in C#, you know what type each element is in your data structure. Without generics, you would have no idea what type each element was.
+The last useful thing about the runtime knowing the type of your generic is a better debugging experience. When you're debugging a generic in C#, you know what type each element is in your data structure. Without generics, you would have no idea what type each element was.
```

Comment:
Again unrelated to your PR:
- "The last useful thing" --> a horrible phrase. Perhaps "An additional advantage"